shapely/geometry/multipolygon.py

```diff
@@ -55,11 +55,7 @@ def __new__(self, polygons=None):
             return polygons
 
         polygons = getattr(polygons, "geoms", polygons)
-        polygons = [
-            p
-            for p in polygons
-            if p and not (isinstance(p, polygon.Polygon) and p.is_empty)
-        ]
+        polygons = [p for p in polygons if p]
```

Comment:
Let's just add a small comment about why we are doing this (now it is less explicit)
```suggestion
        # remove None and empty polygons from list of Polygons
        polygons = [p for p in polygons if p]
```